src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String value;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.